.github/scripts/doc_update/update_docs.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+import os
+from dirsync import sync
+import yaml
+import argparse
+
+parser = argparse.ArgumentParser(description="Keptn Documentation Updater")
+parser.add_argument('--version', '-v', help='Keptn LT Version', default="development", required=True, dest='version')
+parser.add_argument('--update-main', '-u', action='store_true', help='Update main version', dest='update_main')
+parser.add_argument('--klt-repo', '-k', help='Keptn LT Repo Path', required=True, dest='klt_repo')
+parser.add_argument('--klt-docs', '-d', help='Keptn Docs Repo Path', required=True, dest='klt_docs')
+parser.add_argument('--klt-examples', '-e', help='Keptn Examples Repo Path', required=True, dest='klt_examples')
+
+args = parser.parse_args()
+
+klt_repo = args.klt_repo
+klt_docs = args.klt_docs
+klt_examples = args.klt_examples
+version = args.version
+update_main = args.update_main
+
+if klt_docs == "" or klt_repo == "":
+    print("Please provide the path to the Keptn LT and Keptn Docs Repos")
+    exit(1)
+
+# Sync the docs from the KLT repo to the docs folder, sync main-version docs to the root
+sync(klt_repo + '/docs/content/docs', klt_docs + '/content/en/docs-' + version, 'sync', exclude=['^tmp', 'Makefile'], create=True)
```

Comment:
it would be nice to be able to configure the excludes through an external file in the future. maybe in a separate enhancement ticket